Makefile

```diff
@@ -100,49 +101,47 @@ docker-images:
 # Rules for verification, formatting, linting, testing and cleaning #
 #####################################################################
 
-.PHONY: revendor
-revendor:
-	@GO111MODULE=on go mod tidy
-	@GO111MODULE=on go mod vendor
-	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/*
-	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/.ci/*
-	@$(REPO_ROOT)/hack/update-github-templates.sh
-	@ln -sf ../vendor/github.com/gardener/gardener/hack/cherry-pick-pull.sh $(HACK_DIR)/cherry-pick-pull.sh
+.PHONY: tidy
+tidy:
+	@go mod tidy
```

Comment:
@timuthy I had issues running this without ``GO111MODULE=on` like in https://github.com/gardener/gardener-extension-provider-aws/blob/d75666468dda0fe76ff131d91ca24921dccfa578/Makefile#L107C3-L107C14. Could you do a double check again on your side ?